app.server.js

```diff
@@ -11,7 +11,7 @@ const PORT = process.env.PORT || 8080;
 const HOST = "0.0.0.0";
 const GOOGLE_CREDENTIAL =
   process.env.GOOGLE_CREDENTIAL ||
-  "990846956506-bfhbjsu4nl5mvlkngr3tsmfcek24e8t8.apps.googleusercontent.com";
+  "990846956506-bfhbjsu4nl5mvlkngr3tsmfcek24e8t8.apps.googleusercontent.com"; 
```

Comment:
Removes trailing 

```suggestion
  "990846956506-bfhbjsu4nl5mvlkngr3tsmfcek24e8t8.apps.googleusercontent.com";
```

Protip: You can install editor config plugin (https://editorconfig.org/) in your text editor that prevent this.